float32 throttle
int16 steer
int16 swivel
